paddle/fluid/operators/detail/brpc_server.cc

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/operators/detail/brpc_server.h"
+#include "paddle/fluid/operators/detail/request_handler.h"
+
+namespace sendrecv {
+class BRPCServiceImpl : public SendRecvService {
+ public:
+  BRPCServiceImpl(const std::unordered_map<
```

Comment:
Can use `typedef std::unordered_map<std::string, paddle::operators::detail::RequestHandler*> HandlerMap` to simplify the code.

Reply:
Done. Thanks!